### DISCUSSION TITLE
duplicate images displayed

### DISCUSSION THREAD

**Discussion by oison111** [containerd/nerdctl#1306] [Q&A] [answered]
Why are duplicate images displayed????

![image](https://user-images.githubusercontent.com/102855127/183668399-77b296ac-7e5f-4aac-b2c9-f9cfbe0ec5c6.png)

![image](https://user-images.githubusercontent.com/102855127/183594586-f1e0a512-402d-4432-989a-725adce14ae5.png)
![image](https://user-images.githubusercontent.com/102855127/183594835-e7580e9e-57d5-46a5-8f7d-b8bdae1dbb91.png)
![image](https://user-images.githubusercontent.com/102855127/183606049-c9906a8d-56a9-46f3-95eb-6c2666429b36.png)

**Comment by junnplus** [comment, marked as answer]
This is expected.

Pulling an image in the `k8s.io` namespace will [create 3 references](https://github.com/containerd/containerd/blob/2b661b890f689716b2928408e386e656d8dd7118/pkg/cri/server/image_pull.go#L176):
1. imageID
2. repoTag
3. repoDigest
